### DISCUSSION TITLE
多个节点时，想选中底层级的节点，应该怎么做。

### DISCUSSION THREAD

**Discussion by douyalzd** [antvis/X6#4522] [Q&A]
多个节点时，想选中低层级的节点，应该怎么做，有没有好方法处理
如图：输入框层级是2，白块的层级是1，当选中输入框后，就无法选中白块了
有没有什么办法点击穿透选中白块的方法
<img width="363" alt="image" src="https://github.com/user-attachments/assets/62e6c07c-3f8f-4f6d-ac76-872e639709b8">

**Comment by Yeti-xxx** [comment]
你这个类似泳道的功能吧，建议用svg画一个中间镂空的矩形区域，矩形最上方就是input。这样由于中间镂空就可以点击内部的节点了
